x-pack/test/functional/apps/endpoint/host_list.ts

```diff
@@ -67,21 +67,70 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           'xxxx',
         ],
       ];
-      const tableData = await pageObjects.endpoint.getEndpointAppTableData('managementListTable');
+      const tableData = await pageObjects.endpoint.getEndpointAppTableData('hostListTable');
       expect(tableData).to.eql(expectedData);
     });
 
-    it('displays no items found', async () => {
+    it('display details flyout when the hostname is clicked on', async () => {
+      await (await testSubjects.find('hostnameCellLink')).click();
+      await testSubjects.existOrFail('hostDetailsUpperList');
+      await testSubjects.existOrFail('hostDetailsLowerList');
+    });
+
+    it('displays no items found when empty', async () => {
       // clear out the data and reload the page
       await esArchiver.unload('endpoint/metadata/api_feature');
-      await pageObjects.common.navigateToUrlWithBrowserHistory('endpoint', '/management');
+      await pageObjects.common.navigateToUrlWithBrowserHistory('endpoint', '/hosts');
       // get the table data and verify no entries appear
-      const tableData = await pageObjects.endpoint.getEndpointAppTableData('managementListTable');
+      const tableData = await pageObjects.endpoint.getEndpointAppTableData('hostListTable');
       expect(tableData[1][0]).to.equal('No items found');
       // reload the data so the other tests continue to pass
       await esArchiver.load('endpoint/metadata/api_feature');
     });
 
+    describe('has a url with a host id', () => {
+      before(async () => {
+        await pageObjects.common.navigateToUrlWithBrowserHistory(
+          'endpoint',
+          '/hosts',
+          'selected_host=cbe80003-6964-4e0f-aba1-f94c32b44e95'
```

Comment:
ideally values like this wouldn't be hardcoded so the tests aren't so coupled to the particular es_archive